test/e2e_node/topology_manager_test.go

```diff
@@ -680,7 +680,7 @@ func teardownSRIOVConfigOrFail(ctx context.Context, f *framework.Framework, sd *
 }
 
 func runTMScopeResourceAlignmentTestSuite(ctx context.Context, f *framework.Framework, configMap *v1.ConfigMap, reservedSystemCPUs, policy string, numaNodes, coreCount int) {
-	threadsPerCore := getSMTLevel()
+	threadsPerCore := smtLevelFromSysFS()
```

Comment:
```suggestion
	smtLevel := smtLevelFromSysFS()
```

Above will requires also update in line 717

```
numCores := smtLevel * coreCount
```

Reasoning for improved consistency, proposed to use one common variable name to return value as done in cpu_manager_test.go and cpu_manager_metrics_test.go

Reply:
right call about consistency, changing as suggested